lib/client/beans.html

```diff
@@ -3,6 +3,7 @@
     <link rel="stylesheet" href="css/beans.css">
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
     <script src="./js/ui/base/Control.js"></script>
+    <script src="./js/ui/controls/Menu.js"></script>
 </head>
```

Comment:
My bad - I forgot to pass this your way:

```
<script src="./js/ui/Menu.js"></script>
    <script>
        function renderView() {
            var beansView = new BeansView();
            $("#beans").appendChild(beansView.render());
        }

        window.onload = renderView;
    </script>
```

We'll also want to remove the Menu related html from this file since we're rendering it on the JS side now. :)